coverage/include-directory: drop the unbundled headers from the baseline.

zeek and Spicy installed their vendored copies of nlohmann_json, prometheus-cpp,
expected-lite, doctest and sqlite3 under the include prefix; with the system
packages supplying those headers, nothing installs them and the 84 entries are
gone. Test data only -- Spicy's JIT resolves them from the include prefix.

--- a/testing/btest/Baseline/coverage.include-directory/..include-dir.out
+++ b/testing/btest/Baseline/coverage.include-directory/..include-dir.out
@@ -354,61 +354,6 @@
 ./hilti/rt/3rdparty/SafeInt/SafeInt.hpp
 ./hilti/rt/3rdparty/SafeInt/safe_math.h
 ./hilti/rt/3rdparty/SafeInt/safe_math_impl.h
-./hilti/rt/3rdparty/nlohmann
-./hilti/rt/3rdparty/nlohmann/adl_serializer.hpp
-./hilti/rt/3rdparty/nlohmann/byte_container_with_subtype.hpp
-./hilti/rt/3rdparty/nlohmann/detail
-./hilti/rt/3rdparty/nlohmann/detail/abi_macros.hpp
-./hilti/rt/3rdparty/nlohmann/detail/conversions
-./hilti/rt/3rdparty/nlohmann/detail/conversions/from_json.hpp
-./hilti/rt/3rdparty/nlohmann/detail/conversions/to_chars.hpp
-./hilti/rt/3rdparty/nlohmann/detail/conversions/to_json.hpp
-./hilti/rt/3rdparty/nlohmann/detail/exceptions.hpp
-./hilti/rt/3rdparty/nlohmann/detail/hash.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input
-./hilti/rt/3rdparty/nlohmann/detail/input/binary_reader.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input/input_adapters.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input/json_sax.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input/lexer.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input/parser.hpp
-./hilti/rt/3rdparty/nlohmann/detail/input/position_t.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators
-./hilti/rt/3rdparty/nlohmann/detail/iterators/internal_iterator.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators/iter_impl.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators/iteration_proxy.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators/iterator_traits.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators/json_reverse_iterator.hpp
-./hilti/rt/3rdparty/nlohmann/detail/iterators/primitive_iterator.hpp
-./hilti/rt/3rdparty/nlohmann/detail/json_custom_base_class.hpp
-./hilti/rt/3rdparty/nlohmann/detail/json_pointer.hpp
-./hilti/rt/3rdparty/nlohmann/detail/json_ref.hpp
-./hilti/rt/3rdparty/nlohmann/detail/macro_scope.hpp
-./hilti/rt/3rdparty/nlohmann/detail/macro_unscope.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta
-./hilti/rt/3rdparty/nlohmann/detail/meta/call_std
-./hilti/rt/3rdparty/nlohmann/detail/meta/call_std/begin.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/call_std/end.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/cpp_future.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/detected.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/identity_tag.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/is_sax.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/std_fs.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/type_traits.hpp
-./hilti/rt/3rdparty/nlohmann/detail/meta/void_t.hpp
-./hilti/rt/3rdparty/nlohmann/detail/output
-./hilti/rt/3rdparty/nlohmann/detail/output/binary_writer.hpp
-./hilti/rt/3rdparty/nlohmann/detail/output/output_adapters.hpp
-./hilti/rt/3rdparty/nlohmann/detail/output/serializer.hpp
-./hilti/rt/3rdparty/nlohmann/detail/string_concat.hpp
-./hilti/rt/3rdparty/nlohmann/detail/string_escape.hpp
-./hilti/rt/3rdparty/nlohmann/detail/value_t.hpp
-./hilti/rt/3rdparty/nlohmann/json.hpp
-./hilti/rt/3rdparty/nlohmann/json_fwd.hpp
-./hilti/rt/3rdparty/nlohmann/ordered_map.hpp
-./hilti/rt/3rdparty/nlohmann/thirdparty
-./hilti/rt/3rdparty/nlohmann/thirdparty/hedley
-./hilti/rt/3rdparty/nlohmann/thirdparty/hedley/hedley.hpp
-./hilti/rt/3rdparty/nlohmann/thirdparty/hedley/hedley_undef.hpp
 ./hilti/rt/3rdparty/tinyformat
 ./hilti/rt/3rdparty/tinyformat/tinyformat.h
 ./hilti/rt/any.h
@@ -604,38 +549,9 @@
 ./zeek/3rdparty
 ./zeek/3rdparty/ConvertUTF.h
 ./zeek/3rdparty/bsd-getopt-long.h
-./zeek/3rdparty/doctest.h
 ./zeek/3rdparty/modp_numtoa.h
-./zeek/3rdparty/nonstd
-./zeek/3rdparty/nonstd/expected.hpp
 ./zeek/3rdparty/patricia.h
-./zeek/3rdparty/prometheus-cpp
-./zeek/3rdparty/prometheus-cpp/include
-./zeek/3rdparty/prometheus-cpp/include/prometheus
-./zeek/3rdparty/prometheus-cpp/include/prometheus/check_names.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/client_metric.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/collectable.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/counter.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/builder.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/ckms_quantiles.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/core_export.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/future_std.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/time_window_quantiles.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/detail/utils.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/family.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/gauge.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/histogram.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/info.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/labels.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/metric_family.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/metric_type.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/registry.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/serializer.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/summary.h
-./zeek/3rdparty/prometheus-cpp/include/prometheus/text_serializer.h
 ./zeek/3rdparty/setsignal.h
-./zeek/3rdparty/sqlite3.h
 ./zeek/3rdparty/zeek_inet_ntop.h
 ./zeek/Anon.h
 ./zeek/Attr.h
